articles/communication-services/concepts/email/email-overview.md

```diff
@@ -46,7 +46,7 @@ Key features include:
 
 * [Get started with create and manage Email Communication Service in Azure Communication Service](../../quickstarts/email/create-email-communication-resource.md)
 
-* [Get started by connecting Email Communication Service with a Azure Communication Service resource](../../quickstarts/email/connect-email-communication-resource.md)
+* [Get started by connecting Email Communication Service with Azure Communication Service resource](../../quickstarts/email/connect-email-communication-resource.md)
```

Comment:
```suggestion
* [Get started by connecting Email Communication Service with an Azure Communication Service resource](../../quickstarts/email/connect-email-communication-resource.md)
```
Adding an article (an vs. a) because the sentence is talking about an Azure Communication Service *resource*.